src/Console/Report/FixReport/GitlabReporter.php

```diff
@@ -39,15 +52,22 @@ public function generate(ReportSummary $reportSummary): string
     {
         $report = [];
         foreach ($reportSummary->getChanged() as $fileName => $change) {
+            $diffs = $this->diffParser->parse($change['diff']);
+            $firstChunk = isset($diffs[0]) ? $diffs[0]->getChunks() : [];
+            $firstChunk = array_shift($firstChunk);
             foreach ($change['appliedFixers'] as $fixerName) {
                 $report[] = [
-                    'description' => $fixerName,
+                    'check_name' => $fixerName,
+                    'description' => isset($this->fixers[$fixerName]) ?
+                        $this->fixers[$fixerName]->getDefinition()->getSummary() : '',
+                    'categories' => ['Style'],
                     'fingerprint' => md5($fileName.$fixerName),
                     'severity' => 'minor',
                     'location' => [
                         'path' => $fileName,
                         'lines' => [
-                            'begin' => 0, // line numbers are required in the format, but not available to reports
+                            'begin' => $firstChunk ? $firstChunk->getStart() : 0,
```

Comment:
Can the logical condition before `?` be explicitly boolean? Like `$firstChunk instanceof Something`?